cocos/core/animation/marionette/graph-eval.ts

```diff
@@ -1248,8 +1259,8 @@ export class MotionStateEval extends StateEval {
         return stateStatus;
     }
 
-    public resetToPort () {
-        this._toPort.progress = 0.0;
+    public resetToPort (at: number) {
+        this._toPort.progress = at;
```

Comment:
should take some actions when 'at' is  unlegal data

Reply:
The valid input is non-negative finite numbers for now. I think it's safe to use it without assertion.